src/lib/vault/getFormSubmission.ts

```diff
@@ -0,0 +1,40 @@
+import { GetCommand } from "@aws-sdk/lib-dynamodb";
+import { dynamodbClient } from "@lib/awsServicesConnector";
+
+interface FormSubmission {
+  content: string;
+}
+
+export async function getFormSubmission(
+  formId: string,
+  submissionName: string,
+): Promise<FormSubmission | undefined> {
+  try {
+    const response = await dynamodbClient.send(
+      new GetCommand({
+        // biome-ignore lint/style/useNamingConvention: <explanation>
```

Comment:
Biome does not yet support multiline ignore

Reply:
I feel we can even ignore the whole rule (style/useNamingConvention)

Reply:
Been thinking about it but then, even though it may not happen a lot, we basically lose that check entirely 😕 .
We could keep it for now and use the explanation placeholder to explain why we ignore it. If it starts being very annoying then let's remove it

Reply:
Make sense! Will disable for now ;)

Reply:
Found a middle ground and set the severity to `info` for that specific rule so that we can still see that something is not normal (underline in blue instead of yellow) but it won't impact the lint check result.